### PR TITLE
PP-30-create-header-and-navbar

### PR DESCRIPTION
**Pull request by Victoria240**
Not sure if there are any other links we need to include in the navbar, but I tried to follow what we had in the design mocks
 I've also incorporated the login and signup feature

### CODE REVIEW

File: lib/mongo/dbConnect.js [hunk 1]
```diff
@@ -27,12 +27,12 @@ async function dbConnect() {
       return mongoose;
     });
   }
-  try {
-    cached.conn = await cached.promise;
-  } catch (e) {
-    cached.promise = null;
-    throw e;
-  }
+  // try {
```
Comment (AnnaPestova1) [inline lib/mongo/dbConnect.js]: please remove commented lines.

**Review by AnnaPestova1 — CHANGES_REQUESTED** [review]
Hi Victoria! You did a great job! Your header looks nice!
I mentioned couple issues:
1. seems you didn't connected MongoDB on your side. Follow this instruction https://docs.google.com/document/d/12ol8Vwz4sGetn3YBDVqjiInk9u6008QfmLFmAtTlxAs/edit?usp=sharing and, please, comment out all lines you changed in dbConnect.js file.
2. Our team decided use MUI as our styling system. Could you please change you code according MUI documentation (https://mui.com/material-ui/getting-started/).
Thank you!

**Review by AnnaSolovykh — CHANGES_REQUESTED** [review]
Nice work! The only thing is that when a person is logged in, the "log in" link in the navbar should turn to "log out". This feature is in AuthControl component, you need to import it nicely into Navbar and style accordingly. In AuthControl there is mainly the logic, not styled. Screenshot 2024-03-19 at 13 04 01 The functionality we need to preserve from dev-main: Screenshot 2024-03-19 at 13 09 13

**Review by AnnaPestova1 — CHANGES_REQUESTED** [review]
Hi Victoria! the navbar looks great! could you please solve errors in console? 
image

maybe this example will help you to figure out how to solve warnings: https://mui.com/material-ui/react-app-bar/

**Review by AnnaSolovykh — APPROVED** [review]

**Review by AnnaPestova1 — APPROVED** [review]

**Pull request merged**